### PR TITLE
Rewrite the template to starter

### PR DESCRIPTION
**Pull request by JiriLojda**
- Prepare a couple of useful functions and hooks
- Create a context component that will initialize the custom element, put useful info into React context and handle the height of the element
- Adjust the readme to guide through the process of starting a custom element from this starter

### Motivation

Which issue does this fix? Fixes #`issue number`

If no issue exists, what is the fix or new feature? Were there any reasons to fix/implement things that are not obvious?

### Checklist

- [ ] Code follows coding conventions held in this repo
- [ ] Automated tests have been added
- [ ] Tests are passing
- [ ] Docs have been updated (if applicable)
- [ ] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

If manual testing is required, what are the steps?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,109 +7,125 @@
 [![Discord][discord-shield]][discord-url]
 
 
-# Kontent.ai React Integration Template
+# Kontent.ai React Custom Element Starter
 
-This template can be used to jumpstart your own custom element development with Kontent.ai. It contains all the necessary tools for creating a new [Custom Element](https://kontent.ai/learn/tutorials/develop-apps/integrate/content-editing-extensions/), a UI extension for content editors.
```
Comment (IvanKiral) [inline README.md:12]: nitpick: link to Kontent.ai :D

File: README.md [hunk 2]
```diff
@@ -7,109 +7,125 @@
...
+You can specify whether they should select only one or several.
+The function returns details of the selected items.
+
+### promptToSelectAssets
+
+Use this function to prompt the user to select assets.
+You can specify whether they should select only one or several and whether they should only select images or any asset.
+The function returns details of the selected assets.
+
+## Running the project
```
Comment (IvanKiral) [inline README.md]: NITPICK: My personal preference would be having this more to the top.. Less skilled developers who are not working so much with `package.json` might find this information useful more times when developing. Ultimately, the decision is on you

File: src/IntegrationApp.tsx [hunk 3]
```diff
@@ -1,119 +1,54 @@
...
+        selectedItemNames,
+        elementValue,
+        setElementValue,
+        isDisabled,
+        config,
+        projectId,
+        item,
+        variant,
+        watchedElementsValues,
+        selectAssets,
```
Comment (IvanKiral) [inline src/IntegrationApp.tsx]: Why not add atleast one button with selectAssets? :) it showcases how to use it and might help to some newbies developers :)

**Review by IvanKiral — CHANGES_REQUESTED** [review]

File: src/IntegrationApp.tsx (hunk 3 above)
Comment (JiriLojda) [reply src/IntegrationApp.tsx]: I wanted the app to be as clean as possible, but a small example might be worth it. I will try to add something.

File: README.md (hunk 2 above)
Comment (JiriLojda) [reply README.md]: What do you mean by "this"? The "Utilities in this repository" section?

File: README.md (hunk 1 above)
Comment (JiriLojda) [reply README.md:12]: No sure what you mean 😅

File: README.md (hunk 2 above)
Comment (IvanKiral) [reply README.md]: I meant runing the project

File: README.md (hunk 1 above)
Comment (IvanKiral) [reply README.md:12]: i am sorry i have forgotten to remove this comment

**Review by IvanKiral — APPROVED** [review]

**Pull request merged**